src/scripts.js

```diff
@@ -143,6 +143,35 @@ function dropYear(dates) {
     }
   });
 
+  //User Hydration Input
+  $('.hydration-submit').click(hydrationHandler);
+
+  function hydrationHandler() {
+    let dateInput = $('#user-hydration-date').val();
+    let ounceInput = Number($('#user-hydration-oz').val());
+
+    fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData', {
```

Comment:
This looks really good! The fetch looks clean and that was smart just making the object and using the inputs in the POST. Also good thinking on adding the message within that then - that looks real good. I wonder if we will make a function for the error states and then put that into a then block or a catch block??